hack/fetch-releases.sh

```diff
@@ -1,86 +1,113 @@
-#!/bin/bash
+#!/usr/bin/env sh
```

Comment:
if we are going to user `sh`
then we need update the if conditions
`[[ ]]` is not supported I guess

Reply:
yep ci showing error for this

Reply:
```
#!/usr/bin/env bash
```